src/manifest.json

```diff
@@ -11,22 +12,46 @@
   },
   "description": "The Bitcoin Lightning wallet for direct payments across the globe, Bitcoin Lightning applications and passwordless logins.",
   "homepage_url": "https://getAlby.com/",
+
   "web_accessible_resources": [
     "js/inpageScript.bundle.js",
     "js/inpageScriptWebLN.bundle.js",
     "js/inpageScriptNostr.bundle.js"
   ],
+
+  "__chrome__web_accessible_resources": [
+    {
+      "resources": [
+        "js/inpageScript.bundle.js",
+        "js/inpageScriptWebLN.bundle.js",
+        "js/inpageScriptNostr.bundle.js"
+      ],
+      "matches": ["https://*/*"]
+    }
+  ],
+
   "permissions": [
```

Comment:
if we have `__chrome__permissions` shouldn't those then be firefox__permission? or do those get overwritten?